src/server/rest/v1/service/UserService.ts

```diff
@@ -483,6 +486,10 @@ export default class UserService {
               // Set default value
               user.importedBy = importedBy;
               user.importedOn = importedOn;
+              user.importedData = {
+                'autoActivateUserAtImport' : UtilsSecurity.filterBoolean(req.headers.autoactivateuseratimport),
```

Comment:
[Low] Same case in header on variable names ?

Reply:
thanks to nodejs